chromadb/config.py

```diff
@@ -76,6 +76,7 @@
     "chromadb.auth.ServerAuthenticationProvider": "chroma_server_authn_provider",
     "chromadb.auth.ServerAuthorizationProvider": "chroma_server_authz_provider",
     "chromadb.db.system.SysDB": "chroma_sysdb_impl",
+    "chromadb.execution.executor.abstract.Executor": "chroma_executor_impl",
```

Comment:
"abstract" being in the path here feels a bit odd. As now the config path has abstract in it?